packages/core/src/guards.ts

```diff
@@ -163,10 +163,10 @@ function checkNot(
 }
 
 /**
```

Comment:
Seems like the Prettier output changed some whitespace throughout the document for some reason.  Go to the next comment for the actual updates 👇